premailer/premailer.py

```diff
@@ -132,33 +167,64 @@ def __init__(self, html, base_url=None,
                  disable_basic_attributes=None,
                  disable_validation=False,
                  cache_css_parsing=True,
+                 cache_css_parsing_size=1000,
+                 cache_css_output=True,
+                 cache_css_output_size=1000,
                  cssutils_logging_handler=None,
                  cssutils_logging_level=None,
                  disable_leftover_css=False,
                  align_floating_images=True,
                  remove_unset_properties=True):
+
+        '''
+        Initializes the premailer class.
+
+        TODO: document the rest of options
+
+        base_url: Specifies base URL for loading external stylesheets via
+            relative URLs.
+
+            If base_url is True, premailer will transform all URLs by
+            joining them with the base_url.
+
+        preserve_internal_links: If specified, the URL transforming behavior
+            enabled by setting base_url to True will be disabled for links to
+            named anchors.
+
+        preserve_inline_attachments: If specified, the URL transforming
+            behavior enabled by setting base_url to True will be disabled for
+            any links with cid: scheme.
+
+        disable_link_rewrites: If specified, the URL transforming behavior
+            enabled by setting base_url to True will be disabled altogether.
+
+        keep_style_tags: Specifies whether to delete the <style> tag once
+            it's been processed. If True, all original css will be
+            preserved.
+
+        include_star_selectors: Specifies whether to process or ignore
+            selectors like '* { foo:bar; }'
+
+        cache_css_parsing: Specifies whether to cache the CSS parsing results.
+
+        cache_css_parsing_size: Specifies the size for various CSS parsing
+            caches. If set to None, the size of the caches will not be limited.
+
+        cache_css_output: Specifies whether to cache the CSS output results.
```

Comment:
Unlike the `cache_*_size` options, this one is actually *not* operational. Let's keep this. Setting this *will* potentially have an effect on the generated HTML output.